3.component_model.md

```diff
@@ -216,16 +216,17 @@ While it is not required, it is RECOMMENDED that `image` names be suffixed with
 
 > The name field is required and must be 63 characters or less, beginning and ending with an alphanumeric character ([a-z0-9A-Z]) with dashes (-), underscores (_), dots (.), and alphanumerics between.
 
-### WorkloadSetting
+### Properties
 
-This section describes additional configuration for a workload (settings that are not about a container runtime).
+A properties object (for workloadSettings, trait and scope configuration) is an object whose structure is determined by the workload type, trait or scope property schema. It may be a simple value, or it may be a complex object.
```

Comment:
Could we combine this with the section in  _7.workload_types.md_?

Reply:
I changed the section in `7.workload_types.md`